cmd/output/output.go

```diff
@@ -24,13 +24,17 @@ import (
 type OutputFormat struct {
 	// JsonPath specifies a subpath of the output to extract data from
 	JsonPath string
+	Yaml     bool
 }
```

Comment:
Instead of just saving these flags, could you instead do something like this

```
type OutputConverter interface {
  convert(input interface{}) (string, error)
}
```

And then put the three different `OutputFormat` implementations in this file.

Reply:
I'm not sure I follow. Why do we need multiple?

Reply:
Sorry, I meant three different `OutputConverters`... one for YAML, one for plain JSON, and one for JSON with a `JsonPath`. (Or maybe those last two can be the same?) i.e. so change this:
```
func ParseOutputFormat(outputFormat string) (*OutputFormat, error)
```
to
```
func CreateOutputConverter(outputFormat string) (*OutputConverter, error)
```

Reply:
I refactored this to be 3 different implementations of `type OutputFormater func(interface{}) ([]byte, error)` instead of a single implementation with if statements.